src/fsharp/FSComp.txt

```diff
@@ -256,7 +256,7 @@ chkUnionCaseDefaultAugmentation,"default augmentation of the union case"
 438,chkDuplicateMethod,"Duplicate method. The method '%s' has the same name and signature as another method in this type."
 438,chkDuplicateMethodWithSuffix,"Duplicate method. The method '%s' has the same name and signature as another method in this type once tuples, functions, units of measure and/or provided types are erased."
 439,chkDuplicateMethodCurried,"The method '%s' has curried arguments but has the same name as another method in this type. Methods with curried arguments cannot be overloaded. Consider using a method taking tupled arguments."
-440,chkCurriedMethodsCantHaveOutParams,"Methods with curried arguments cannot declare 'out', 'ParamArray', 'optional', 'ReflectedDefinition' or 'byref' arguments"
+440,chkCurriedMethodsCantHaveOutParams,"Methods with curried arguments cannot declare 'out', 'ParamArray', 'optional', 'ReflectedDefinition', 'byref', 'CallerLineNumber', or 'CallerFilePath' arguments"
```

Comment:
Ah yes, hadn't realized that.  Tricky since in some situations there isn't even a valid name, e.g. a script that just calls one of these members.  I suppose we'd add it to TcEnv and set it within each function definition.